README.md

```diff
@@ -1,38 +1,83 @@
-# Valkyrie stubs
+# Valkyrie Stubs
 
-Stubs for valkyrie integration testing. Integration tests for |Valkyrie](https://github.com/valkyrie-fnd/valkyrie) use this library to cut out the dependency to a PAM. 
+[![](https://img.shields.io/badge/License-MIT%20-brightgreen.svg)](./LICENSE.md)
+[![](https://img.shields.io/github/actions/workflow/status/valkyrie-fnd/valkyrie-stubs/gh-workflow.yml)](https://github.com/valkyrie-fnd/valkyrie-stubs/actions/workflows/gh-workflow.yml)
+![](https://img.shields.io/github/last-commit/valkyrie-fnd/valkyrie-stubs)
+[![](https://img.shields.io/website?url=https%3A%2F%2Fvalkyrie.bet)](https://valkyrie.bet/docs)
+![](https://img.shields.io/github/go-mod/go-version/valkyrie-fnd/valkyrie-stubs)
+![](https://img.shields.io/github/languages/top/valkyrie-fnd/valkyrie-stubs)
+![](https://img.shields.io/tokei/lines/github/valkyrie-fnd/valkyrie-stubs)
+![](https://img.shields.io/maintenance/yes/2023)
 
-## Running standalone
-Another option is to start both `Valkyrie` and `Valkyrie-stubs` in standalone mode in order to 
-test some provider integration. 
+## A testing library for the Valkyrie iGaming Aggregator
+
+This repository contains testing stubs and utilities for [Valkyrie](https://github.com/valkyrie-fnd/valkyrie), an 
+open source iGaming Aggregator.
+
+This testing repository includes:
+
+* **genericpam** - an implementation of Valkyrie's Player Account Management (PAM) API, for more information please read
+  the [documentation](https://valkyrie.bet/docs/wallet/valkyrie-pam/valkyrie-pam-api)
+* **memorydatastore** - a datastore implementation for **genericpam** persisted in memory, useful for building integration 
+  tests
+* **backdoors** - HTTP APIs for creating sessions and other testing data fixtures in genericpam
+* **broken** - an HTTP API and a web interface for simulating various failure scenarios (timeouts, errors, etc)
+
+## Building
+
+The project is built from source by running:
+
+```bash
+go build
+```
+
+## Running
+
+The project can run standalone using the built binary:
+
+```bash
+./valkyrie-stubs
+```
+
+There is also a `docker-compose.yml` available which will start **valkyrie** together with **valkyrie-stubs** 
+acting as PAM:
 
-To start standalone instances of Valkyrie with stubs using a memory database:
 ```bash
-git clone git@github.com:valkyrie-fnd/valkyrie-stubs.git
-cd valkyrie-stubs
```

Comment:
Perhaps state that all actions and tips in this article are executed from valkyrie-stubs root folder?